src/libraries/System.Net.WebSockets.Client/src/System/Net/WebSockets/BrowserWebSockets/BrowserWebSocket.cs

```diff
@@ -236,6 +227,37 @@ internal async Task ConnectAsyncJavaScript(Uri uri, CancellationToken cancellati
             }
         }
 
+
+        private void onCloseCallback(JSObject? closeEvt, CancellationToken cancellationToken)
```

Comment:
Nit: Can we use PascalCasing for members like this?

Reply:
Addressed in recent commits